src/charm.py

```diff
@@ -127,6 +142,32 @@ def flask_layer(self) -> dict:
             },
         }
 
+    def _on_statsd_prometheus_exporter_pebble_ready(self, _event: PebbleReadyEvent) -> None:
+        """Handle the statsd-prometheus-exporter-pebble-ready event."""
+        statsd_container = self.unit.get_container("statsd-prometheus-exporter")
+        statsd_layer = {
+            "summary": "statsd exporter layer",
+            "description": "statsd exporter layer",
+            "services": {
+                "statsd-prometheus-exporter": {
+                    "override": "replace",
+                    "summary": "statsd exporter service",
+                    "user": "nobody",
+                    "command": "/bin/statsd_exporter",
+                    "startup": "enabled",
+                }
+            },
+            "checks": {
```

Comment:
Do you think that should go to the flask module (webserver)?
The same comment was made [here](https://github.com/canonical/github-actions-exporter-operator/pull/3#discussion_r1193309642).

Reply:
Please see the discussion here: https://github.com/canonical/flask-k8s-operator/pull/13#discussion_r1198574412
Thanks!